openfl-workspace/flower-app-pytorch/src/grpc/connector/flower/local_grpc_server.py

```diff
@@ -0,0 +1,141 @@
+import logging
+import threading
+import queue
+import grpc
+from concurrent.futures import ThreadPoolExecutor
+from flwr.proto import grpcadapter_pb2_grpc
+from src.grpc.connector.flower.message_conversion import flower_to_openfl_message, openfl_to_flower_message
+from multiprocessing import cpu_count
+import signal
+import psutil
+import time
+
+logger = logging.getLogger(__name__)
+
+
+class LocalGRPCServer(grpcadapter_pb2_grpc.GrpcAdapterServicer):
```

Comment:
I think `FlowerInteropServer` better describes what this component actually does. WDYT?